### PR TITLE
Volunteering offer fixes

### PR DESCRIPTION
**Pull request by abrabing**

### CODE REVIEW

File: models/volunteering-request-offer-base.js [hunk 1]
```diff
@@ -24,18 +20,22 @@ const VolunteeringRequestOfferBaseSchema = new Schema({
...
+    type: Boolean,
     required: true,
   },
-  notes: {
+  content: {
+    type: mongoose.Schema.Types.ObjectId,
+    ref: 'Content',
+  },
+  regions: {
     type: [String],
```
Comment (OrBin) [inline models/volunteering-request-offer-base.js]: Shouldn't it be `[Region]`?

File: routes/contents.js [hunk 2]
```diff
@@ -1,7 +1,7 @@
 const Router = require('koa-router');
 const Content = require('../models/content');
 const DBMethods = require('./base-db-methods');
-const Category = require('../models/category');
+const Category = require('../models/category').default;
```
Comment (OrBin) [inline routes/contents.js]: Doesn't it work without `.default`? I'm pretty sure it should.

**Review by OrBin — CHANGES_REQUESTED** [review]

**Comment by OrBin** [comment]
Also, fix the build error. It has an error when compiling the typescript code: https://travis-ci.org/CommuniBee/CommuniBee-Front-End/jobs/488525371#L536

File: models/volunteering-request-offer-base.js (hunk 1 above)
Comment (abrabing) [reply models/volunteering-request-offer-base.js]: It should be subregion

File: models/volunteering-request-offer-base.js [hunk 3]
```diff
@@ -29,7 +29,7 @@ const VolunteeringRequestOfferBaseSchema = new mongoose.Schema({
     ref: 'Content',
   },
   regions: {
-    type: [String],
+    type: ['SubRegion'],
```
Comment (OrBin) [inline models/volunteering-request-offer-base.js]: Does it work like that?
From a short search (and the little exprience I've had with mongoose), I think it should look like:
```js
regions: {
  type: [
    {
      type: Schema.ObjectId,
      ref: 'Choices'
    }
  ],
  required: true,
},
```

**Review by OrBin — CHANGES_REQUESTED** [review]

**Review by OrBin — APPROVED** [review]

**Pull request merged**